lib/src/component.dart

```diff
@@ -92,7 +93,7 @@ abstract class FluxComponent<ActionsT, StoresT> extends react.Component {
   ///     redrawOn() => [store.tasks, store.users];
   List<Store> redrawOn() {
     if (store is Store) {
-      return [store];
+      return [store as Store];
```

Comment:
Interesting.. I thought this would have been implied a type cast because of the if statement